test/unit/reducers/__snapshots__/onboarding.spec.js.snap

```diff
@@ -23,7 +23,7 @@ Snapshot Diff:
 +   "isSkipBackupDialogOpen": true,
     "lndConnect": null,
     "name": "",
-    "network": "mainnet",
+    "network": "testnet",
```

Comment:
Looks like this has flipped to `testnet` by mistake. Perhaps you have set `network: 'testnet'` in your local config and are using `local.js` config override file instead of `local-development.js`?